api/controllers/cartController.js

```diff
@@ -2,10 +2,30 @@ const cartService = require("../services/cartService");
 const { asyncErrorHandler } = require("../middleware/errorHandling");
 
 const getCart = asyncErrorHandler(async (req, res) => {
-  const cartList = await cartService.getCart(req.userId, req.query.userId);
+  const cartList = await cartService.getCart(req.userId);
+
   return res.status(200).json({ data: cartList });
 });
 
+const addCart = asyncErrorHandler(async (req, res) => {
+  const addedCart = await cartService.addCart(req.userId, req.body.itemId, req.body.optionId, req.body.quantity);
+  console.log(req)
+  return res.status(201).json({ data: addedCart });
+});
+
+const updateQuantity = asyncErrorHandler(async (req, res) => {
+  const result = await cartService.updateQuantity(req.body.quantity, req.body.itemId, req.userId)
+  return res.status(201).json({ data: result });
+})
```

Comment:
itemId에 대해서 keyerror 추가해 주세요.
```suggestion
const updateQuantity = asyncErrorHandler(async (req, res) => {
  const result = await cartService.updateQuantity(req.body.quantity, req.body.itemId, req.userId)
  return res.status(200).json({ data: result });
})
```